tika-core/src/test/java/org/apache/tika/TikaTest.java

```diff
@@ -62,6 +62,15 @@
 
     protected static Parser AUTO_DETECT_PARSER = new AutoDetectParser();
 
+    /**
+     * Finds a resource with a given name.
+     * @param name name of the desired resource
+     * @return A {@link java.net.URL} object or null
+     */
+    public URL getResource(String name) {
```

Comment:
A more descriptive name - like `getResourceAsUrl()` - would help when reading the code. Wondering if a `getResourceAsUri()` should also be added.

Reply:
Sure. Will update change.